samples/system-test/resource_test.js

```diff
@@ -21,7 +21,7 @@ const tools = require(`@google-cloud/nodejs-repo-tools`);
 const cmd = 'node resource.js';
 const sessionId = require('uuid/v1')();
 
-test.before.serial('Remove all existing resources', async () => {
```

Comment:
Reverted back- issue here had everything to do ava running tests async, and 'removing all existing resources' would happen before that run of serialized tests, not the tests I was adding, which was causing issue- changed to test.before to run before all tests

Reply:
Makes sense

How about using `test.before` instead of `test.after`? `test.before` is generally more correct & less brittle

/cc @ace-n @JustinBeckwith for input here, I'm not intimately familiar with ava and and its syntax for setup/teardown (a) before each test (b) before batches of tests (c) async specific stuff

Reply:
If this doesn't break tests & always works consistently, I no longer consider this a **BLOCKER**

Ideally, I would love to hear from Ace/Justin/someone :)